invenio_app_rdm/config.py

```diff
@@ -333,3 +322,10 @@ def _(x):
 # =============
 INSTANCE_THEME_FILE = './scss/invenio_app_rdm/theme.scss'
 """Instance's theme file path to override the default theme."""
+
+INSTANCE_SEARCH_THEME_FILE = './scss/invenio_app_rdm/search/theme.scss'
+"""Instance's search theme file path to override the default search theme."""
+
+SEARCH_HEADER_TEMPLATE = 'invenio_app_rdm/search/header_search.html'
+
+SEARCH_BASE_TEMPLATE = 'invenio_app_rdm/search/search.html'
```

Comment:
I put this code in a search module here but my intention for the longer run, and I think this is what probably would be done if I am not wrong, is to replace `invenio-search-ui` code with this one. I am not sure if `invenio-rdm-records` should be also responsible for the search but this can be discussed during our call today I guess!

Reply:
I am with Zach on this. I think the search is also not as tight to the records (okay, u need a datamodel/API) but it might be independent of `rdm-records` somehow, its more `invenio-records` and its rest sibling.

I would leave it here for now (It's how it is done in the `cookiecutter-invenio-instance`) and in the long run it will go to `invenio-search-ui`.

Reply:
Got it. Some of it will move to invenio-search-ui: https://github.com/inveniosoftware/invenio-app-rdm/issues/79